opal/mca/threads/pthreads/threads_base_null.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2004-2005 The Trustees of Indiana University and Indiana
+ *                         University Research and Technology
+ *                         Corporation.  All rights reserved.
+ * Copyright (c) 2004-2005 The University of Tennessee and The University
+ *                         of Tennessee Research Foundation.  All rights
+ *                         reserved.
+ * Copyright (c) 2004-2005 High Performance Computing Center Stuttgart,
+ *                         University of Stuttgart.  All rights reserved.
+ * Copyright (c) 2004-2005 The Regents of the University of California.
+ *                         All rights reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#ifndef OPAL_MCA_TIMER_BASE_TIMER_BASE_NULL_H
+#define OPAL_MCA_TIMER_BASE_TIMER_BASE_NULL_H
+
+opal_class_t opal_condition_t_class;
+opal_class_t opal_mutex_t_class;
+opal_class_t opal_recursive_mutex_t_class;
+
+opal_thread_t *opal_thread_get_self(void) {
```

Comment:
This is a `.h` file -- shouldn't all of these functions be marked as `static inline`?

Reply:
They have been fixed; all of such functions in headers are marked as `static inline`.